README.md

```diff
@@ -1,40 +1,39 @@
-# automation firebase project
+# bss-firebase
 
-`firebase emulators:start`
+## Introduction
 
-to deploy one set of functions:
-`firebase deploy --only functions:[namespace]`
+This repo name is a misnomer! What we actually have is a -- er -- _monorepo_ composed of two distinct deployments:
 
-to deploy just the website:
-`firebase deploy --only hosting`
+- [www.bedstuystrong.com](https://bedstuystrong.com/) (`./public/*`)
+- *Cloud Functions for Slack* (`./functions/*`)
+
+## Prerequisites
```

Comment:
these are more internal and not the same as like a CONTRIBUTORS.md, i'd prefer to document them just internally (plus george and i are already working on documenting this process)

Reply:
nw i can remove.